src/e2e-test/features/bigquery/source/BigQuerySourceError.feature

```diff
@@ -27,6 +27,7 @@ Feature: BigQuery source - Validate BigQuery source plugin error scenarios
       | table          | bqIncorrectTableName        |
       | datasetProject | bqIncorrectDatasetProjectId |
 
+  @BigQuery_Source
```

Comment:
Why is this change required? We already have this tag for the entire feature.

Reply:
Hi, Actually the build was failing due to this older test so just added this tag. Reverting it to the previous way now.